src/services/poignArt/syncPoignArtDonationCronJob.ts

```diff
@@ -0,0 +1,98 @@
+import config from '../../config';
+import { logger } from '../../utils/logger';
+import { schedule } from 'node-cron';
+import {
+  Donation,
+  DONATION_STATUS,
+  DONATION_TYPES,
+} from '../../entities/donation';
+import { Project } from '../../entities/project';
+import { convertTimeStampToSeconds } from '../../utils/utils';
+import { getPoignArtWithdrawals, PoignArtWithdrawal } from './api';
+import { fetchGivHistoricPrice } from '../givPriceService';
+import { NETWORK_IDS } from '../../provider';
+
+/**
+ * @see{@link https://github.com/Giveth/impact-graph/issues/433}
+ */
+
+const cronJobTime =
+  (config.get('SYNC_POIGN_ART_CRONJOB_EXPRESSION') as string) || '0 0 * * 0';
+
+export const runSyncPoignArtDonations = () => {
+  logger.debug('runSyncPoignArtDonations() has been called');
+  schedule(cronJobTime, async () => {
+    await importPoignArtDonations();
+  });
+};
+
+const importPoignArtDonations = async () => {
+  try {
+    let startTimestamp = 0;
+    const lastPoignArtDonation = await Donation.createQueryBuilder('donation')
+      .where(`donationType= ${DONATION_TYPES.POIGN_ART}`)
+      .orderBy('createdAt', 'DESC')
+      .getOne();
+    if (lastPoignArtDonation) {
+      startTimestamp = convertTimeStampToSeconds(
+        lastPoignArtDonation.createdAt.getTime(),
+      );
+    }
+    const unchainProjectAddress = process.env
+      .POIGN_ART_RECIPINET_ADDRESS as string;
+    const unchainProject = await Project.createQueryBuilder('project')
+      .where(
+        `"LOWER(fromWalletAddress) = ${unchainProjectAddress.toLowerCase()}`,
+      )
+      .getOne();
+    if (!unchainProject) {
+      logger.info(
+        `importPoignArtDonations() There is no project with walletAddress of ${unchainProjectAddress} in our db`,
+      );
+      return;
+    }
+    const poignArtWithdrawals = await getPoignArtWithdrawals({
+      recipient: unchainProjectAddress,
+      startTimestamp,
+    });
+    for (const poignArtWithdrawal of poignArtWithdrawals) {
+      await createPoignArtDonationInDb(poignArtWithdrawal, unchainProject);
+    }
+  } catch (e) {
+    logger.error('importPoignArtDonations() error', e);
+    throw e;
+  }
+};
+
+const createPoignArtDonationInDb = async (
+  poignArtWithdrawal: PoignArtWithdrawal,
+  unchainProject: Project,
+) => {
+  const ethPrice = (
+    await fetchGivHistoricPrice(poignArtWithdrawal.txHash, NETWORK_IDS.MAIN_NET)
+  ).givPriceInEth;
+  const donation = Donation.create({
+    projectId: unchainProject.id,
+    toWalletAddress: unchainProject.walletAddress,
+    isProjectVerified: unchainProject.verified,
+    fromWalletAddress: 'I should ask it from Lauren',
```

Comment:
Untill this time is not clear what address should we set for poign art donations